### DISCUSSION TITLE
Temperatures

### DISCUSSION THREAD

**Discussion by uusitalo** [valnoxy/t480-oc#81] [Q&A]
I´m having quite hot CPU all the time.
If charger plugged then 74-80 degree C and without 50-60.

How others temperatures are? 

T480 i5-8250U and from bios power saving modes on battery and charging
Ventura 13.2.1

**Comment by cchewanik** [comment]
I have no trouble, my fan barely runs. Check your activity monitor to see if any processes are running at 100%+, I have issues with bluetoothd sometimes

**Comment by trevorwdunn** [comment]
I was having the same issue on mine (processor is i5-7300U). I configured and installed CPUFriend and that solved the cooling issue and seemed to speed up the laptop considerably (probably less thermal throttling; the default fan in the T480 is known to be subpar).
